src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+const populations = document.querySelectorAll('.population');
+
+const populationsArr
+  = [...populations].map(item => +item.innerHTML.replaceAll(',', ''));
+
+const totalPopulation
+  = populationsArr.reduce((accum, population) => accum + population);
```

Comment:
```suggestion
  = populationsArr.reduce((accum, population) => accum + population, 0);
```
it is good practise to set initialValue in any cases